packages/ui-kit/src/Select/components/SelectButton/index.tsx

```diff
@@ -68,22 +73,51 @@ export const SelectButton = React.forwardRef(
       [reset],
     );
 
-    let placeholderContent: ReactNode;
-    if (item) {
-      if (multiselect === MULTISELECT_TYPE_AGGREGATE && item && typeof item === 'string') {
-        placeholderContent = <MultiselectChip onBtnClick={reset}>{item}</MultiselectChip>;
-      } else if (multiselect === MULTISELECT_TYPE_CHIPS && Array.isArray(item)) {
-        placeholderContent = item.map((itm, index) => (
-          <MultiselectChip key={`${itm.value}_${index}`} value={itm.value} onBtnClick={reset}>
-            {itm.title}
-          </MultiselectChip>
-        ));
-      } else {
-        placeholderContent = <span className={style.textContent}>{String(item)}</span>;
+
+    const getPlaceholderAndItem = (): ReactNode => {
+      // Several options here
+      // placeholder space - can be occupied with value, with placeholder(multiple options), or Select children
+      // in case item is selected(presented) - top placeholder should be shown in addition
+      // to the item (on top)
+
+      let itemContent: ReactNode;
+      if (item) {
+        if (multiselect === MULTISELECT_TYPE_AGGREGATE && item && typeof item === 'string') {
```

Comment:
It not very obvious that empty string in JS falsy value, I suggest more clear way
`multiselect === MULTISELECT_TYPE_AGGREGATE && typeof item === 'string' && item.length`